packages/frontend-v2/pages/index.tsx

```diff
@@ -1,23 +1,13 @@
 import type { NextPage } from "next";
-import { SeasonEnum } from "@graduate/common";
+import { Flex, Heading } from "@chakra-ui/react";
 
 const Home: NextPage = () => {
-  // using a type from our common package
-  const seasons: SeasonEnum[] = [
-    SeasonEnum.FL,
-    SeasonEnum.SP,
-    SeasonEnum.S1,
-    SeasonEnum.S2,
-  ];
-
   return (
-    <>
-      <h1>GraduateNU Landing Page:</h1>
-      <h2>Seasons typed using types from our common package!</h2>
-      {seasons.map((season) => (
-        <p key={season}>{season}</p>
-      ))}
-    </>
+    <Flex width="full" justifyContent="center" my="xl">
```

Comment:
seeee we can use things like "<Flex />, and "<Box />" without having to create our styled.divs. These components by default are better than styled.divs cause they have easy access to theme values(spacing, colors, responsiveness). Chakra has a whole suite of layout components too 👀

Reply:
it sounds like we will be defaulting to chakra ui props over emotion/styled then? we should also be careful to not over-use these, (ie multiple nesting of row/column component props and adding lots of nodes over styling)

Reply:
yeap!